app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    commands = command.split()
+    if len(commands) == 3 and commands[0] == "cp":
+        source_file = commands[1]
+        destination_file = commands[2]
```

Comment:
better to use unpacking 
```suggestion
        _, source_file, destination_file = commands
```